packages/typescript/src/components/Interface.tsx

```diff
@@ -87,9 +108,18 @@ export function InterfaceMember(props: InterfaceMemberProps) {
   } else {
     const namer = useTSNamePolicy();
     const name = namer.getName(props.name!, "interface-member");
-    if (props.refkey) {
+    const memberScope = useMemberScope();
+
+    if (memberScope) {
+      const sym = createTSSymbol({
+        name,
+        scope: memberScope.staticMembers as TSOutputScope,
+        refkey: props.refkey,
+        flags: OutputSymbolFlags.StaticMember,
+        tsFlags: TSSymbolFlags.TypeSymbol,
+      });
       return (
-        <MemberDeclaration static name={name} refkey={props.refkey}>
+        <MemberDeclaration static symbol={sym} refkey={props.refkey}>
```

Comment:
don't need to pass refkey here now.

Reply:
Also could probably stand to have the TypeScript equivalent of `MemberDeclaration`.